OpenRA.Game/Game.cs

```diff
@@ -59,6 +59,8 @@ public static class Game
 		static Task discoverNat;
 		static bool takeScreenshot = false;
 
+		public static event Action OnShellMapLoaded;
```

Comment:
Can you please rename this to `OnShellmapLoaded` (little `m`)?  We use "Shellmap" as a single word everywhere except the `LoadShellMap` method.